{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -27,4 +28,4 @@ requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
 build-backend = "poetry_dynamic_versioning.backend"
 
 [tool.poetry.extras]
-docs = ["linkml", "mkdocs-material"]
\ No newline at end of file
+docs = ["linkml", "mkdocs-material"]
```

Comment:
In general you shouldn't need changing any newlines, but this is a typical issue with editor configurations automatically applying this change. Please have a look at your editor configuration with respect to final newlines.

Some editors or beautifiers/linters are adding the final newline, so we would end up having commits adding and removing final newlines.